build/yamls/flow-visibility-e2e.yml

```diff
@@ -0,0 +1,158 @@
+apiVersion: v1
+data:
+  create_table.sh: "#!/usr/bin/env bash\n\n# Copyright 2022 Antrea Authors.\n#\n#
```

Comment:
hopefully this does not change too often as it's a bit messy to update. BTW, is there any way to use a "raw" string here, so that the data is easier to read / troubleshoot / update. Something like this:
```yaml
--- 
apiVersion: v1
data: 
  create_table.sh: |
      # line 1
      # line 2
      echo "hello"
end: true
```
The alternative is to auto-generate this file, which may be the case (maybe there is a script in Theia?), in which case ignore this comment.

Reply:
Thanks Antonin, this file doesn't need to be modified unless we change the fields of flow records exported by Flow Aggregator.
I also thought about auto-generate this file in Theia and downloading it during the e2e test. But I could imagine that when we want to change fields of flow records in the future, we need to change `flow-visibility-e2e.yml` on the Theia side first to let our tests on the Antrea side pass. Not sure whether this is a good approach, may I ask your suggestion pls?

Reply:
I assume that you have a way to auto-generate it manually with a script though, right? based on some files in the theia repo. If the answer is yes, then it's good enough for me and there is no need to change anything.

Reply:
Auto-generate this file from Theia repo sounds good to me. Just to add, create_table.sh content can be formatted well when we using Helm to generate this file as in Theia repo.

Reply:
Yes, we have a script in Theia to generate this manifest. 
Although I made some changes to delete the clickhouse monitor in the generated manifest for now, we will have an update on the script in Theia to generate one exactly same manifest.

Reply:
Hi @antoninbas, we have updated this manifest with a better format and comments about how to generate it, please take a look again, thanks!